novels.sql

```diff
@@ -1,9 +1,9 @@
 CREATE DATABASE greatNovels;
 
-CREATE USER 'novels'@'localhost' IDENTIFIED
-WITH mysql_native_password BY 'n0v31s!';
+CREATE USER 'Library'@'localhost' IDENTIFIED
+WITH mysql_native_password BY 'B@@ks!';
 
-GRANT ALL ON greatNovels.* TO 'novels'@'localhost';
+GRANT ALL ON greatNovels.* TO 'Library'@'localhost';
 
 USE greatNovels;
```

Comment:
You can delete this file. We only used .sql files as a stand in until we learned migrations. In a real project you would have migrations from the beginning and would never have a .sql at all.